.github/workflows/hooks-warnings.yml

```diff
@@ -0,0 +1,65 @@
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  comment:
+    if: ${{ github.event.issue.pull_request }}
```

Comment:
Should the job also be limited on the comment author? Should this run in the comment is not from @conan-center-bot ?

Reply:
Yes, I'm not doing it currently,  in order to be able to do some manual tests